Core/source/core/pgp-msg.ts

```diff
@@ -301,19 +301,18 @@ export class PgpMsg {
   private static cryptoMsgGetSignedBy = async (msg: OpenpgpMsgOrCleartext, keys: SortedKeysForDecrypt) => {
     keys.signedBy = Value.arr.unique(await PgpKey.longids(msg.getSigningKeyIds ? msg.getSigningKeyIds() : []));
     if (keys.signedBy.length && typeof Store.dbContactGet === 'function') {
-      const verificationContacts = await Store.dbContactGet(undefined, keys.signedBy);
-      keys.verificationContacts = verificationContacts.filter(contact => contact && contact.pubkey) as Contact[];
+      const signerContacts = (await Store.dbContactGet(undefined, keys.signedBy))
+        .filter(contact => contact && contact.pubkey) as Contact[];
```

Comment:
I overlooked this. Anything using `dbContactGet` won't work on iOS because that method is not implemented (and will remain that way). 

you could pass `verificationPubkeys` also into `cryptoMsgGetSignedBy`, and then I suppose you can remove the if/else in `getSortedKeys`. So `cryptoMsgGetSignedBy` would fill both `keys.signedBy` (ids) and `keys.forVerification` (the supplied ones). Then since pubkeys are supplied, you remove the `dbContactGet` here, and also the `typeof Store.dbContactGet === 'function'`.

Reply:
Yes, I suppose you can simplify to let it go through cryptoMsgGetSignedBy and pass verificationPubkeys in